core/tests/test_views.py

```diff
@@ -117,6 +164,61 @@ def test_add_comment_unauthorized(client: Client, post: Post) -> None:
     assert reverse("login") in response.url
 
 
+def test_reported_list_post_by_admin(client: Client, admin: User, post: Post, report_data: dict) -> None:
+    client.force_login(admin)
+    response = client.post(reverse("post-report", kwargs={"pk": post.pk}), data=report_data)
+    assert response.status_code == 302
+    assert reverse("home") in response.url
+    response = client.get(reverse("post-list-reported"))
+    assert response.status_code == 200
+    reports_count = PostReport.objects.filter(verified=False).count()
+    assert reports_count > 0
+
+
+def test_reported_list_post_by_user(client: Client, user: User, post: Post, report_data: dict) -> None:
+    client.force_login(user)
+    response = client.post(reverse("post-report", kwargs={"pk": post.pk}), data=report_data)
+    assert response.status_code == 302
+    response = client.get(reverse("post-list-reported"))
+    assert response.status_code == 302
+    assert reverse("home") in response.url
+
+
+def test_reported_list_post_by_anonymous_user(client: Client) -> None:
+    response = client.get(reverse("post-list-reported"))
+    assert response.status_code == 302
+    assert "/users/login/?next=/core/reported-posts/" in response.url
+
+
+def test_reported_detail_post_by_user(client: Client, user: User, post: Post, report_data: dict) -> None:
+    client.force_login(user)
+    response = client.post(reverse("post-report", kwargs={"pk": post.pk}), data=report_data)
+    assert response.status_code == 302
+    response = client.get(reverse("reported-post", kwargs={"pk": 1}))
+    assert response.status_code == 403
+
+
+def test_reported_detail_post_by_anonymous_user(client: Client, post_report: PostReport) -> None:
+    response = client.get(reverse("post-report", kwargs={"pk": post_report.pk}))
+    assert response.status_code == 302
+    assert f"/users/login/?next=/core/post/report/{post_report.pk}/" in response.url
```

Comment:
Please, use reverse not hardcoded urls. If we change url structure this test wil fail, which is bad. 
Simple would be to check if this is login url from settings. And there should be error message that you need to log in. 
So I would check if `settings.LOGIN_URL in response.url and response.url.endswith /core/post/report/{post_report.pk}/`

Reply:
Pseudocode of course.

Reply:
Done